build.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+SOURCE_DIR=$(readlink -f "${BASH_SOURCE[0]}")
+cd "$(dirname "$SOURCE_DIR")" || exit
```

Comment:
This should fail with an error code if it does fail.

Reply:
i hope error code '1' is good enough